skyrl-train/skyrl_train/config/ppo_base_config.yaml

```
@@ -166,6 +171,10 @@ trainer:
   logger: "wandb"
   dump_data_batch: false
   dump_eval_results: true
+  rope_scaling:
```

Could we add a comment that this supports all parameters in https://huggingface.co/docs/transformers/main/en/internal/rope_utils#transformers.RopeParameters? 

```suggestion
  rope_scaling: # supports all parameters in https://huggingface.co/docs/transformers/main/en/internal/rope_utils#transformers.RopeParameters
```

---

Oh I see now that the `RopeParameters` class is only introduced in transformers 4.57.1, which we upgraded to only recently after #528